packages/sdk/package.json

```diff
@@ -35,6 +35,7 @@
     "bignumber.js": "^9.1.2",
     "bip32": "^4.0.0",
     "bip39": "^3.1.0",
+    "bitcoin-address-validation": "^2.2.3",
```

Comment:
Not really. This is a popular library, same as bitcoinlib.js. I think what we're trying to avoid is re-create functions that have been created and tested on other frameworks. This is also used on Ordzaar repo btw - https://github.com/ruigomeseu/bitcoin-address-validation

Reply:
@thedoublejay There are unnecessary catches created by this library. Please check my other review comments. 

One example: https://github.com/ordzaar/ordit-sdk/pull/14/files#diff-20cd5ff2a77269557b1fa9fdd947bf69eec25e86598acafd690dc4bbd19a32d9R37

Reply:
We have been keeping the dependencies for Ordit-SDK as small as possible because it deals w/ private keys and sensitive data. No doubt the package in question is a popular library but my point is: have we or somebody else audited it? I guess its a different case for Ordzaar because Ordzaar does not deal directly w/ the private keys.

From my understanding, some things like address validation can surely be done on our side. We don't need a dependency just for a simple validation that exists anyway.

Reply:
The issue with the simple validation is that it is _not_ exactly simple, because right now the regex we are using doesn't determine whether it's really a valid address, so we introduced this dependency.

I understand there are catches (particularly for regtest) and the concern about security. I looked at the code for that library and didn't notice any rogue dependencies/postinstall scripts. For this case, we can pin this dependency's version to prevent any instances of supply chain attacks in NPM's ecosystem. For any other new bitcoin libraries, I agree that we should avoid importing them until they are audited.

Reply:
Yep I have reviewed it since we're using it on Ordzaar. It's a pretty small library and contains bunch of tests. The idea is we don't want to create new tests to recreate these functionalities. The library catches all scenarios and also is well tested. There are only two files (ts and type), you can take a look.

Reply:
Okay, I will audit this package. Another concern that I have is unnecessary `unknown` address format introduced because of this package. `unknown` would introduce too many bugs when we finally migrate all modules to this repo. It'd be good if we can stick to the original formats.